api/api_sources/schema-migration-sql/CountrySchema/CountrySchema.sql

```diff
@@ -0,0 +1,35 @@
+-- ### Creating Table: country ### --
+
+        
+CREATE TABLE country ();
+ALTER TABLE country ADD COLUMN country_code VARCHAR(3) PRIMARY KEY;
+ALTER TABLE country ADD COLUMN description VARCHAR(100) NULL;
```

Comment:
The "description" column should be NOT NULL as there should always be a description.

Reply:
Resolved